tests.toml

```diff
@@ -0,0 +1,15 @@
+test_format = 1.0
+
+[default]
+
+    # -------------------------------
+    # Default args to use for install
+    # -------------------------------
+
+    args.default_home_server="matrix.org"
```

Comment:
```suggestion
```

This is not needed because this is the same default value as in the manifest :sweat_smile: